### PR TITLE
Add angular project

### PR DESCRIPTION
**Pull request by blacksam07**
## Summary
- Install Angular CLI
- Create angular project

## Trello card
[3-angular-cli](https://trello.com/c/YhVOGmJy/3-angular-cli)

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    "no-outputs-metadata-property": true,
+    "template-banana-in-box": true,
+    "template-no-negated-async": true,
+    "use-lifecycle-interface": true,
+    "use-pipe-transform-interface": true
+  },
+  "rulesDirectory": [
+    "codelyzer"
+  ]
+}
```
Comment (betoobernal) [inline tslint.json]: add space to the end

File: e2e/protractor.conf.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    defaultTimeoutInterval: 30000,
+    print: function() {}
+  },
+  onPrepare() {
+    require('ts-node').register({
+      project: require('path').join(__dirname, './tsconfig.json')
+    });
+    jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
+  }
+};
```
Comment (betoobernal) [inline e2e/protractor.conf.js]: same

File: angular.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+          }
+        }
+      }
+    }
+  },
+  "defaultProject": "wolox-books",
+  "cli": {
+    "analytics": "65cc7ddb-befc-4734-852b-644d9768bd8f"
+  }
+}
```
Comment (betoobernal) [inline angular.json]: same

File: tslint.json (hunk 1 above)
Comment (blacksam07) [reply tslint.json]: auto-generated by CLI

File: e2e/protractor.conf.js (hunk 2 above)
Comment (blacksam07) [reply e2e/protractor.conf.js]: auto-generate

File: e2e/src/app.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { AppPage } from './app.po';
```
Comment (CrisVega08) [inline e2e/src/app.e2e-spec.ts]: Delete this file, you are not going to do test

File: src/app/app.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
+import { TestBed, async } from '@angular/core/testing';
```
Comment (CrisVega08) [inline src/app/app.component.spec.ts]: Same delete

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: src/app/app.module.ts [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    AppRoutingModule
+  ],
+  providers: [],
```
Comment (betoobernal) [inline src/app/app.module.ts]: if it's empty is better to remove it.

**Review by CrisVega08 — APPROVED** [review]

**Review by betoobernal — APPROVED** [review]

**Pull request merged**